docs/webapi/scrollIntoView.mustache

```diff
@@ -0,0 +1,8 @@
+Scroll element into viewport.
+
+```js
+I.scrollIntoView('#submit');
```

Comment:
it's better to add the examples with using params

Reply:
thanks @PeterNgTr - added